modules/web/src/de/diedavids/cuba/dataimport/web/importfile/ImportDataImportConfigurationMatchException.groovy

```diff
@@ -1,8 +1,11 @@
 package de.diedavids.cuba.dataimport.web.importfile
 
+import de.diedavids.cuba.dataimport.dto.ColumnValidationResult
+
 class ImportDataImportConfigurationMatchException extends RuntimeException {
 
-    ImportDataImportConfigurationMatchException() {
-        super('Import File does not match import configuration')
+    ImportDataImportConfigurationMatchException(ColumnValidationResult result) {
+        super('Import File does not match import configuration. Please check required columns: '
```

Comment:
I would store the `ColumnValidationResult` variable here as an attribute in order to use it in the frontend to have a translation and pass those fields in as a parameter.